UPGRADING.rst

```diff
@@ -4,79 +4,156 @@ Upgrading to Graylog 4.0.x
 
 .. _upgrade-from-33-to-40:
 
-Deprecation of cluster stats endpoints
-======================================
+.. contents:: Overview
+   :depth: 3
+   :backlinks: top
 
-Starting with v4.0, the cluster stats endpoints are deprecated and will be removed in a future version. Those include:
+Breaking Changes
+================
+
+Changes to the Elasticsearch Support
+------------------------------------
+
+Starting with Graylog v4.0, bigger changes to the Elasticsearch versions supported are happening:
+
+  - Support for Elasticsearch versions prior to v6.8.0 are dropped.
```

Comment:
I know I am correcting myself, but it should be `is dropped`, not `are dropped` :)

Reply:
@dennisoelkers Thanks, I pushed a fix!